src/BloomBrowserUI/bookLayout/basePage.less

```diff
@@ -257,17 +257,27 @@ preview.css.
 //currently, in edit mode every page looks like page 1 to css, so left-vs-right margins are just confusing
 //so we only turn them on when making PDF
 .publishMode :not(.outsideFrontCover):not(.outsideBackCover){
-  &.bloom-page:nth-of-type(odd) .marginBox  {
-	  left: @MarginOuter;
-  }
-  &.bloom-page:nth-of-type(even) .marginBox  {
-	  left: @MarginInner;
-  }
+	&.bloom-page:nth-of-type(odd){
```

Comment:
But of course :-) Alternatively, I often don't try to exclude the special cases, as that makes it hard to read. Instead, you can make use of the 2 rules that would make your special case win: it is more specific, and it comes later.

So you could just have 

    .marginBox { left: @MarginOuter; }

followed by your new rule:

    // override that for the special cases of A5Landscape and A4Landscape. A6Landscape is not included here because....